### DISCUSSION TITLE
Welcome to xontrib-powerline3 Discussions!

### DISCUSSION THREAD

**Discussion by jnoortheen** [jnoortheen/xontrib-powerline3#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pannet1** [comment]
i am not able to get the icons right. can you help me install the correct font to install the missing icon

![image](https://github.com/jnoortheen/xontrib-powerline3/assets/12350276/6075fe71-05b5-45b2-be3f-a27400e5ca88)
